tests/e2e/metrics/test_metrics.py

```diff
@@ -1,40 +1,69 @@
-import datetime
-import uuid
+import json
 
-import boto3
 import pytest
-from e2e import conftest
-from e2e.utils import helpers
 
+from tests.e2e.utils import helpers
 
-@pytest.fixture(scope="module")
-def config() -> conftest.LambdaConfig:
-    return {
-        "parameters": {},
-        "environment_variables": {
-            "POWERTOOLS_METRICS_NAMESPACE": "powertools-e2e-metric",
-            "POWERTOOLS_SERVICE_NAME": "test-powertools-service",
-            "METRIC_NAME": f"business-metric-{str(uuid.uuid4()).replace('-','_')}",
-        },
-    }
 
+@pytest.fixture
```

Comment:
Shouldn’t this fixtures go to shared pytest config file? I can easily imagine we gonna reuse them in every test group we have so far (metric, tracing, logging)

Reply:
Nope, because we learned the hard way how [this grows](https://github.com/heitorlessa/aws-lambda-powertools-python/blob/chore/refactor-e2e/tests/functional/idempotency/test_idempotency.py#L47) and Pytest fixtures dependency injection makes it hard to trace where a fixture comes from.

This is a trade-off where we're keeping the absolutely minimum number of fixtures as close to the tests as possible (I can CTRL+F).